_inc/client/plans-prompt/index.jsx

```diff
@@ -58,4 +63,9 @@ export class PlansPrompt extends React.Component {
 	}
 }
 
-export default PlansPrompt;
+export default connect(
+	state => ( {
+		plans: getAvailablePlans( state ),
+	} ),
```

Comment:
No need for the comma if we remove the next line: 
```suggestion
	} )
```